airtable/.env

```diff
@@ -1,7 +1,24 @@
-# Your Airtable API Key, Base ID and Base table name
+# description: Your Airtable API Key
+# format: secret
+# link: https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-
+# required: true
 AIRTABLE_APIKEY=keyAbcD12efG3HijK
+
+# description: Your Airtable Base ID
+# format: text
+# required: true
```

Comment:
should this have a link as well? I'm just wondering about consistency since like `chat-token` or `video-token` has the same link twice for example on each key